semgrep-core/src/tainting/Dataflow_tainting.ml

```diff
@@ -903,9 +1018,10 @@ let (fixpoint :
     | None -> Lval_env.empty
     | Some in_env -> in_env
   in
+  let top_sinks = top_level_sinks_in_nodes config flow in
```

Comment:
I don't completely understand why we need to initialize this. Would appreciate if you explained! (It might not need a comment, I just want to understand and perhaps I'm a little tired)